docs/blog/newsletter-november-2021.rst

```diff
@@ -0,0 +1,137 @@
+
+--------------------------------------------------
+Documentation gets respect (or at least attention)
+--------------------------------------------------
+
+Two recent reports highlight the increased attention documentation seems to be getting in the worlds of software development.
+
+The `2021 State of DevOps report from Google Cloud <https://cloud.google.com/devops/state-of-devops>`__ includes data showing the effects of docs quality on devops team performance, and describes practices to improve the quality of your documentation. The report is based on seven years of research and data from more than 32,000 professional worldwide, focusing on "capabilities and practices that drive software delivery, operation, and organizational performance." So the focus is not on docs -- but the report includes good documentation as one of six findings characteristic of the 26% of teams studied who fall into the highest performing category. This is _internal_ documentation, and its quality has a statistically significant impact on how secure, reliable, and effective a team's devops practices are. Quality documentation efforts should include critical use cases for products and services, guidelines for updating and editing existing documentation, clearly defined ownership and responsibility for documentation, inclusion of docs as part of the software development process, and recognition of docs work during performance reviews and promotions.
+
+The `State of Software Quality API 2021 report <https://smartbear.com/state-of-software-quality/api/>`__ from SmartBear similarly points to "accurate and detailed documentation" as a key aspect of the overall success of an API, second only to "ease of use" for API consumers. Disappointingly few survey respondents rated their API docs as "Very Good" -- but only 16% of teams include technical writers for their API docs. On the other hand, more than half the respondents contribute to API docs in some way, and 60-70% of respondents work for companies that include a formal API documentation process -- but only about half of those companies consider documentation a priority. Respondents cited a range of obstacles to providing quality documentation, but they all fell into the larger category of insufficient resources -- time, tools, personnel, budget, more. On another front, our documentarians who so diligently work with OpenAPI definitions will be happy to read that OpenAPI usage continues high, but GraphQL and AsyncAPI continue to grow in popularity.
```

Comment:
Should it be "...budget, and more" rather than "...budget, more" in the third-to-last line?

Reply:
yes it should!